### DISCUSSION TITLE
can I move my fosstodon account to self hosted instance

### DISCUSSION THREAD

**Discussion by AbhimanyuAryan** [mastodon/mastodon#19512] [Q&A] [answered]
Hi, I am moving from Twitter to Mastodon my long term goal is to host my data on personal server and not use public severs. So my question is can I move all my followers, following tweets to personal instance/server from fosstodon?

Like a pointer say:

x user is following my account on fosstodon. So the reference doesn't break he can see my tweets and still remains my follower but I can migrate to another server/instance and nothing breaks

**Comment by Gargron** [comment, marked as answer]
Yes to followers/following. You keep that when moving. Not posts though.

**Reply by AbhimanyuAryan** [reply]
Thanks
